hooks/pre/30-upgrade.rb

```diff
@@ -102,6 +102,36 @@ def mark_qpid_cert_for_update
   end
 end
 
+# rubocop:disable MethodLength
+# rubocop:disable Style/MultipleComparison
+def upgrade_qpid_paths
+  qpid_dir = '/var/lib/qpidd'
+  qpid_data_dir = "#{qpid_dir}/.qpidd"
```

Comment:
Going to address this with a downstream MR since upstream is only supporting EL7